docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md

```diff
@@ -18,7 +18,7 @@ If this is the first time you've deployed the AWS Observability solution, read t
 
 1. Sign on to the AWS Management console.
 1. Choose an option:
```

Comment:
Can we write why the user needs to choose the option?

Reply:
updated